extensions/strategies/ta_stoch_bollinger/strategy.js

```diff
@@ -40,50 +40,54 @@ module.exports = {
       then(function(inbol){
         ta_stoch(s, 'stoch',  s.options.stoch_periods, s.options.stoch_k, s.options.stoch_k_ma_type, s.options.stoch_d, s.options.stoch_d_ma_type).
           then(function(inres) {
-    
             if (!inres) return cb()
             var divergent = inres.k[inres.k.length-1] - inres.d[inres.k.length-1]
             s.period.stoch_D = inres.d[inres.d.length-1]
             s.period.stoch_K = inres.k[inres.k.length-1]
             var last_divergent = inres.k[inres.k.length-2] - inres.d[inres.d.length-2]
             var _switch = 0
-            var nextdivergent = (( divergent + last_divergent ) /2) + (divergent - last_divergent) 
-            if ((last_divergent <= 0 && (divergent > 0)) ) _switch = 1 // price rising 
-            if ((last_divergent >= 0 && (divergent < 0)) ) _switch = -1 // price falling
+            var nextdivergent = ((divergent + last_divergent)/2) + (divergent - last_divergent) 
+            if (last_divergent <= 0 && divergent > 0) {
+				_switch = 1 // price rising
+			}
```

Comment:
might need to format to match the rest of the file here

Reply:
OK, applied VS Code beautifier and updated the switch color indicator

Reply:
Thanks!